### PR TITLE
Fixes weird unicode rendering in station announcements

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

e.g ' gets replaced with \&#39; and all that

## Why It's Good For The Game

it's a fix

## Changelog
:cl:
fix: fixed weird announcement text renderings
/:cl:

### CODE REVIEW

**Comment by utilbird** [comment]
Couldn't you just use `html_decode`?

**Comment by jupyterkat** [comment]
decoding it allows players to manipulate the html tags and we don't want that

**Comment by jupyterkat** [comment]
does not work yet, don't merge

**Comment by ghost** [comment]
> does not work yet, don't merge

It does work, at least it does on TGMC where I just ported it to. 
Proof:
https://github.com/tgstation/TerraGov-Marine-Corps/pull/5897
image

**Comment by jupyterkat** [comment]
doesn't work on announcement messages made by request consoles though

**Comment by francinum** [comment]
holy shit was it really this fucking simple

**Comment by jupyterkat** [comment]
yes, utf-8

**Comment by jupyterkat** [comment]
image
working now

File: code/__HELPERS/priority_announce.dm [hunk 1]
```diff
@@ -57,7 +57,8 @@
 
 	for(var/mob/M in GLOB.player_list)
 		if(!isnewplayer(M) && M.can_hear())
-			var/complete_msg = "<span class='big bold'><font color = red>[html_encode(title)]</font color><BR>[html_encode(message)]</span><BR>"
+			var/complete_msg = "<meta charset='UTF-8'>"
+			complete_msg += "<span class='big bold'><font color = red>[html_encode(title)]</font color><BR>[html_encode(message)]</span><BR>"
```
Comment (utilbird) [inline code/__HELPERS/priority_announce.dm]: May as well just combine them

**Review by utilbird — APPROVED** [review]

**Pull request merged**